src/SAWScript/Crucible/LLVM/Override.hs

```diff
@@ -660,6 +662,45 @@ executeCond opts sc cc cs ss = do
             (ss ^. MS.csFreshPointers)
   OM (setupValueSub %= Map.union ptrs)
 
+  liftIO . mapM_ print $ cs ^. MS.csPreState . MS.csAllocs
+
+  sym <- use syminterface
+  mem <- readGlobal . Crucible.llvmMemVar $ cc ^. ccLLVMContext
+  sub <- use setupValueSub
+
+  let mutableAllocs = Map.filter (view isMut) $ cs ^. MS.csPreState . MS.csAllocs
+      allocPtrs =
+        (\(ptr, spec) ->
+           ( ptr
+           , _allocSpecBytes spec
+           , "state of memory allocated in precondition not described in postcondition"
+           )
+        ) <$> Map.elems (Map.intersectionWith (,) sub mutableAllocs)
+      LLVMModule _ _ mtrans = cc ^. ccLLVMModule
+      gimap = Crucible.globalInitMap mtrans
+      mutableGlobals = cs ^. MS.csGlobalAllocs
+
+  globalPtrs <- liftIO . fmap catMaybes . forM mutableGlobals $ \(LLVMAllocGlobal _ s@(L.Symbol st)) ->
+    case Map.lookup s gimap of
+      Just (_, Right (mt, _)) -> do
+        ptr <- Crucible.doResolveGlobal sym mem s
+        pure $ Just
+          ( ptr
+          , Crucible.memTypeSize (Crucible.llvmDataLayout ?lc) mt
+          , mconcat [ "state of mutable global variable \""
+                    , pack st
+                    , "\" not described in postcondition"
+                    ]
+          )
+      _ -> pure Nothing
+
+  mem' <- foldM (\m (ptr, sz, msg) ->
```

Comment:
Instead of a custom tuple type here, you could reuse `LLVMAllocSpec`. In fact, it might be worth considering replacing `AllocGlobal` with `AllocSpec` and reusing code like `doAlloc`/`executeAllocation`.